HISTORY.rst

```diff
@@ -11,6 +11,7 @@ XXXX-XX-XX
   ``/proc`` pseudo files line by line. This should help having more consistent
   results.
 - 2057_, [OpenBSD]: add support for `cpu_freq()`_.
+- 2039_, 2099_: drop support for Python 2.6, EOL since 2013-10-2.
```

Comment:
please `git pull` your branch; there is an entry for this a couple of lines earlier:

```
- 1053_: dropped support for Python 2.6.  (patch by Matthieu Darbois)
- ```

Feel free to add your name (also to CREDITS file).

Reply:
I don't see this at https://github.com/giampaolo/psutil/blob/master/HISTORY.rst, maybe you need to push it?

Reply:
@hugovk sorry you right; I forgot to push indeed hehel. I pushed now.